attestation-service/Cargo.toml

```diff
@@ -61,7 +61,9 @@ tokio.workspace = true
 tonic = { workspace = true, optional = true }
 uuid = { version = "1.1.2", features = ["v4"] }
 veraison-apiclient = { git = "https://github.com/chendave/rust-apiclient", branch = "token", optional = true }
+ear = { git = "https://github.com/chendave/rust-ear", branch = "test" }
```

Comment:
need to sync with upstream once my change in the fork have been merged.

Reply:
two of them still pending: https://github.com/veraison/rust-ear/pulls?q=is%3Apr+author%3Achendave+

Reply:
@Xynnn007 this was updated.